freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/termination/action/StackTerminationFinishedAction.java

```diff
@@ -23,13 +25,18 @@ public class StackTerminationFinishedAction extends AbstractStackTerminationActi
     @Inject
     private Set<AbstractConfigRegister> configRegisters;
 
+    @Inject
+    private AuthDistributorService authDistributorService;
+
     public StackTerminationFinishedAction() {
         super(TerminateStackResult.class);
     }
 
     @Override
     protected void doExecute(StackTerminationContext context, TerminateStackResult payload, Map<Object, Object> variables) {
-        configRegisters.forEach(configProvider -> configProvider.delete(context.getStack()));
+        Stack stack = context.getStack();
```

Comment:
A log would be nice to have here :)

Reply:
What log? This is only a new method call, which has logs.

Reply:
Yeah, but this is an action and iterates over on multiple config providers, from debugging point of view it would be easier to deal with if we can have fundamental logs from this class.

Reply:
I have added the log, but if it is important, then it would be better to somehow enforce logging in doExecute methods, because 90% of doExecute methods lack logging.